api/namex/resources/payment/utils.py

```diff
@@ -139,7 +141,52 @@ def merge_payment_request(nr_model, config=None):
                 'country': country,
                 'postalCode': postal_code
             }
-        }
+        },
+        'details': build_payment_details(nr_model)
     }
 
     return payment_request
+
+def build_payment_details(nr_model):
+    """Build payment details."""
+    details = []
+    details.append(
+        {
+            'label': 'NR Number:',
+            'value': nr_model.nrNum
+        }
+    )
+    details.append(
+        {
+            'label': 'Name Choices:',
+            'value': ''
+        }
+    )
+    option1 = re.search('\|1(.*)1\|', nr_model.nameSearch)
```

Comment:
I don't think you should use the nameSearch column for this since that is specifically there for the advanced search and may change if we have time to try making it faster. Is there a reason you aren't using the names model? (i.e. nr_model.names.all() will give you all the names it has with choice/name/etc. info

Reply:
Yes, you're right. Fixed that. Thanks :)